arangosh/Import/QuickHistogram.h

```diff
@@ -246,18 +252,24 @@ class QuickHistogram : public arangodb::Thread {
 class QuickHistogramTimer {
  public:
   explicit QuickHistogramTimer(QuickHistogram& histo)
-      : _intervalStart(std::chrono::steady_clock::now()), _histogram(histo) {}
+    : _intervalStart(std::chrono::steady_clock::now()), _histogram(histo),
+    _objects(1) {}
```

Comment:
does this need to be initialized to 1 or to 0?

Reply:
Default was one to cover cases where count is not being maintained.  It implies one measurement ... which maybe means one object.